app/assets/stylesheets/modules/full_text_highlight.scss

```diff
@@ -3,10 +3,8 @@ dt.prepared-search-container {
   color: $gray-dark;
   cursor: pointer;
   display: inline-block;
-  float: none;
   text-align: initial;
   width: auto;
-  @extend .col-12;
```

Comment:
This already has .col-3 on the element and since we have `width: auto` this adds nothing.